### PR TITLE
yadm: initial commit

### PR DESCRIPTION
**Pull request by kosciak9**
provides [yadm](https://github.com/TheLocehiliosan/yadm), dotfiles manager. closes issue #768. only change to the source code - fixing shebang, as yadm is pure bash.

### CODE REVIEW

File: packages/yadm/build.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+TERMUX_PKG_BUILD_IN_SRC="yes"
+TERMUX_PKG_DEPENDS="git"
+
+termux_step_make() {
+	:
+}
+
+termux_step_make_install () {
+    sed -i -E "1s@^#\!(.*)/[sx]?bin/(.*)@#\!/data/data/com.termux/files/usr/bin/\2@" $TERMUX_PKG_SRCDIR/yadm
+	cp $TERMUX_PKG_SRCDIR/yadm $TERMUX_PREFIX/bin/yadm
```
Comment (Grimler91) [inline packages/yadm/build.sh]: You can omit the sed'ding, build-package.sh already does this, see https://github.com/termux/termux-packages/blob/2d93ba823e31b358926ff6ecd80975fc687244c2/build-package.sh#L1121

File: packages/yadm/build.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/TheLocehiliosan/yadm
+TERMUX_PKG_DESCRIPTION="Yet Another Dotfiles Manager"
+TERMUX_PKG_VERSION=1.12.0
+TERMUX_PKG_REVISION=1
```
Comment (Grimler91) [inline packages/yadm/build.sh]: You can remove this line, TERMUX_PKG_REVISION defaults to 0

File: packages/yadm/build.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	cp $TERMUX_PKG_SRCDIR/yadm $TERMUX_PREFIX/bin/yadm
+
+	# Install the yadm.1 man page:
+	mkdir -p $TERMUX_PREFIX/share/man/man1/
+	cp $TERMUX_PKG_SRCDIR/yadm.1 $TERMUX_PREFIX/share/man/man1/
+}
+
+termux_step_post_massage () {
+    :
+}
```
Comment (Grimler91) [inline packages/yadm/build.sh]: You don't need to override termux_step_post_massage

**Comment by kosciak9** [comment]
I have fixed lines you mentioned

**Comment by Grimler91** [comment]
Great! Could you perhaps also use tabs for the indentation? Most build.sh scripts use tab instead of four spaces

**Comment by kosciak9** [comment]
Sure, I will get it done tomorrow. 

On 11 September 2018 22:00:32 CEST, Henrik Grimler <notifications@github.com> wrote:
>Great! Could you perhaps also use tabs for the indentation? Most
>build.sh scripts use tab instead of four spaces
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/termux/termux-packages/pull/2825#issuecomment-420404958

---- 
pozdrawiam, 
Franciszek Madej

**Comment by kosciak9** [comment]
I have changed indent to tabs. Also switched out comments for something more verbose. Looking forward to comments and eventually merging.

**Comment by rickard-von-essen** [comment]
Would be awesome to get this in.

**Comment by xeffyr** [comment]
Implemented in [unstable-packages](https://github.com/termux/unstable-packages) repository.
```
pkg install unstable-repo
pkg install yadm
```

**Pull request closed without merging**